frappe/integrations/doctype/s3_backup_settings/s3_backup_settings.py

```diff
@@ -35,11 +35,20 @@ def validate(self):
 			frappe.throw(_("Invalid Access Key ID or Secret Access Key."))
 
 		try:
-			conn.create_bucket(Bucket=bucket_lower, CreateBucketConfiguration={
-				'LocationConstraint': self.region})
-		except ClientError:
-			frappe.throw(_("Unable to create bucket: {0}. Change it to a more unique name.").format(bucket_lower))
-
+			response = conn.head_bucket(Bucket=bucket_lower)
+			# The operation returns a 200 OK if the bucket exists and you have permission to access it.
+			bucket_name_exist = True
+		except ClientError as e:
+			# If a client error is thrown, then check that it was a 403 error or 404 error.
+			error_code = e.response['Error']['Code']
+			if error_code == '403':
+				frappe.throw(_(f"403 Forbidden. Do not have permission to access this bucket: {bucket_lower}"))
```

Comment:
```suggestion
				frappe.throw(_("403 Forbidden. Do not have permission to access {0} bucket.").format(bucket_lower))
```

Reply:
Hi hrwX,

Could you please let me know the reason why we use `str.format()` instead of `f"{}"`.
To be backwards compatible with earlier Python versions?

Thank you.

Reply:
Andy, it isn't really necessary that frappe will always be run with Python3. And since Literal String Interpolation wasn't really introduced up until [PEP-0498](https://www.python.org/dev/peps/pep-0498), which got put in Python3.6, this code won't work on Python versions lower than that.

Reply:
Got it, thanks for the explanation.